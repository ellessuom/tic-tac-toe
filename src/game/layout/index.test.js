import React from 'react';
import { shallow } from 'enzyme';
import Layout from './index';

describe('<Layout />', () => {
  let wrapper;
  beforeAll(() => {
    wrapper = shallow(
      <Layout>
        <div className="layout-test" />
      </Layout>
    );
  });

  it('should have rendered single child', () => {
    expect(wrapper.children()).toHaveLength(1);
  });

  it('should render layout structure correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render layout with correct test child', () => {
    expect(wrapper.contains(<div className="layout-test" />)).toBe(true);
  });

});
