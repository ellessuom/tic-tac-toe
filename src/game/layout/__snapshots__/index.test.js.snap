// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render layout structure correctly 1`] = `
<div
  className="game-layout"
>
  <div
    className="layout-test"
  />
</div>
`;
